app/main.py

```diff
@@ -1,3 +1,78 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
Other can be int and float too. Fix everywhere
```suggestion
    def __add__(self, other: "Distance" | int | float) -> "Distance":
```